pkg/converter/pod_converter.go

```diff
@@ -27,7 +27,11 @@ func (p *podConverter) Convert(k8sObj interface{}) (interface{}, error) {
 	endpoint := api.NewWorkloadEndpoint()
 
 	endpoint.Metadata.Workload = fmt.Sprintf("%s.%s", pod.Namespace, pod.Name)
-	endpoint.Metadata.Labels = pod.ObjectMeta.Labels
+	if pod.ObjectMeta.Labels != nil {
+		endpoint.Metadata.Labels = pod.ObjectMeta.Labels
+	} else {
+		endpoint.Metadata.Labels = make(map[string]string)
```

Comment:
If we're not specifying a size, I'd prefer to avoid using `make` (it's not a typical pattern in the rest of the Calico code).

so:

```go
endpoint.Metadata.Labes = map[string]string{}
```